mk/tests.mk

```diff
@@ -9,15 +9,22 @@ CACHE_OBJS := $(addprefix $(CACHE_BUILD_DIR)/, $(CACHE_OBJS))
 OBJS += $(CACHE_OBJS)
 deps += $(CACHE_OBJS:%.o=%.o.d)
 
-
+ifeq ($(ENABLE_ARC), 1)
+CACHE_CHECK_ELF_FILES := \
+	ARC/cache-new \
```

Comment:
Lowercase should be used for the directory names. i.e., `arc/new` and `arc/put`.